script_automation/esa_ubuntu_k3_single/_index.md

```diff
@@ -0,0 +1,45 @@
+# This example can be used to install ESA on an Ubuntu system with K3s. 
```

Comment:
change this not to be a header - Probably add a header with the title, and that should be just normal text